### PR TITLE
WIP: Try to trigger new distroless build and tests

### PR DESCRIPTION
**Pull request by rikatz**
#### What type of PR is this?

/kind cleanup

#### What this PR does / why we need it:
This PR is a boilerplate to trigger the test on new distroless kproxy images
#### Which issue(s) this PR fixes:
Related to #109406

#### Does this PR introduce a user-facing change?
```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by rikatz** [comment]
/hold
well, apparently the distroless thing is not working as desired 🤦 
Will investigate further if this is something with the image, or something else I need to fix with sig-release

File: test/e2e/network/kube_proxy.go [hunk 1]
```diff
@@ -108,7 +108,7 @@ var _ = common.SIGDescribe("KubeProxy", func() {
 						Name:            "e2e-net-exec",
 						Image:           imageutils.GetE2EImage(imageutils.DebianIptables),
 						ImagePullPolicy: v1.PullIfNotPresent,
-						Command:         []string{"sleep", "600"},
+						Command:         []string{"/bin/sleep", "600"},
```
Comment (aojea) [inline test/e2e/network/kube_proxy.go:111]: Upps ☺️

**Comment by rikatz** [comment]
/close

This PR was just so we could trigger the tests with distroless and keep a historic of changes required to make this works.

I will post all the jobs results in the parent issue

**Pull request closed without merging**